### PR TITLE
added:Hamburger

### PR DESCRIPTION
**Pull request by Dev-AkshatGupta**
Hey everyone added Hamburger in my CL please review it and help me as good as you guys thanks alott Everyone

### CODE REVIEW

File: components/hamburger/hamburger.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
+.menu-btn {
+    /* position: relative; 
+   display: flex; 
+      justify-content: center; 
+      align-items: center; 
+      width: 80px; 
+      height: 80px; 
+      cursor: pointer;  */
+     transition: all .2s ease-in-out;
```
Comment (partha8) [inline components/hamburger/hamburger.css]: Remove the comments :sweat_smile:

**Review by partha8 — APPROVED** [review]
LGTM, good work! You can add transition when the dropdown appears. Do try it. Also have cursor:pointer when we hover over the hamburger menu. Also, reduce the size when you use it. :sweat_smile:

File: components/hamburger/hamburger.css (hunk 1 above)
Comment (Dev-AkshatGupta) [reply components/hamburger/hamburger.css]: Done thanks for reviewing Partha

**Comment by Dev-AkshatGupta** [comment]
> LGTM, good work! You can add transition when the dropdown appears. Do try it. Also have cursor:pointer when we hover over the hamburger menu. Also, reduce the size when you use it. 😅

Actually there needs to be some styles used to over-write depending on the use cases for this actually that's why its like that

File: components/hamburger/hamburger.css [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+   .menu-btn.open .menu-btn__burger::after {
+     transform: rotate(-45deg) translate(35px, 35px);
+   }
+   /* dropdown content */
+   .dropdown {
+       position: relative;
+       display: inline-block;
+     }
+     
+     .dropdown-content {
```
Comment (vrx29) [inline components/hamburger/hamburger.css:48]: please fix this code indentation. Everything else looks good.

**Review by VLeads — APPROVED** [review]
Code is clean. CSS is also good. 💯

**Pull request closed without merging**